libraries/AP_RangeFinder/AP_RangeFinder_GP2Y0E03.cpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (C) 2019  Lucas De Marchi <lucas.de.marchi@gmail.com>
```

Comment:
Incorrect copyright notice :-)

Reply:
I would suggest that you also include the URL of the source you are referring to.
I would like to confirm the license of the referenced source.